Geometry/CMSCommonData/test/runPrint_cfg.py

```diff
@@ -46,9 +46,6 @@
     firstRun = cms.untracked.uint32(1)
 )
 
```

Comment:
@wddgit - could you, please, remove the whole file? It's been replaced by SimG4Core/PrintGeomInfo